packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -161,7 +173,7 @@ async fn get_data_type<P: StorePool + Send>(
     uri: Path<VersionedUri>,
     pool: Extension<Arc<P>>,
 ) -> Result<Json<PersistedDataType>, StatusCode> {
-    read_from_store(pool.as_ref(), &Expression::for_versioned_uri(&uri.0))
+    get_data_types_by_query(pool, Json(Expression::for_versioned_uri(&uri.0)))
```

Comment:
It is quite strange to me to forward to another route implementation rather than calling the underlying thing. Especially since you're rewrapping the Expression in Json

Reply:
Fixed/reverted in 6ad25fd80a13d7d2c39be7c0e75fba0b69963b08